ghost/i18n/locales/ca/comments.json

```diff
@@ -3,69 +3,69 @@
     "{{amount}} comments": "{{amount}} comentaris",
     "{{amount}} hrs ago": "Fa {{amount}} hores",
     "{{amount}} mins ago": "Fa {{amount}} minuts",
-    "{{amount}} more": "",
+    "{{amount}} more": "{{amount}} més",
     "1 comment": "Un comentari",
     "Add comment": "Afegeix un comentari",
     "Add context to your comment, share your name and expertise to foster a healthy discussion.": "Afegeix context al teu comentari, comparteix el teu nom i experiència per fomentar una discussió sana.",
     "Add reply": "Afegeix una resposta",
-    "Already a member?": "Ja ets membre?",
+    "Already a member?": "Ja n'ets membre?",
     "Anonymous": "Anònim",
-    "Are you sure?": "",
+    "Are you sure?": "N'estàs segur?",
     "Become a member of {{publication}} to start commenting.": "Fes-te membre de {{publication}} per començar a comentar.",
     "Become a paid member of {{publication}} to start commenting.": "Fes-te membre de pagament de {{publication}} per començar a comentar",
     "Cancel": "Cancel·lar",
     "Comment": "Comentari",
     "Complete your profile": "Completa el teu perfil",
     "Delete": "Esborrar",
-    "Deleted": "",
+    "Deleted": "Esborrat",
     "Deleted member": "Membre esborrat",
-    "Deleting": "",
+    "Deleting": "Esborrant",
     "Discussion": "Debat",
     "Edit": "Editar",
     "Edit this comment": "Editar aquest comentari",
     "edited": "editat",
     "Enter your name": "Escriu el teu nom",
-    "Expertise": "Conéixement",
+    "Expertise": "Expertesa",
     "Founder @ Acme Inc": "Fundador @ Acme Inc",
     "Full-time parent": "Pare a temps complert",
```

Comment:
Not your change, but should it be complet or complert?

Reply:
Complet!